### DISCUSSION TITLE
Does DS(version3.0+) can run task node alone while workflow is online？| 在工作流是上线状态下可以单独运行其中的任务节点么？

### DISCUSSION THREAD

**Discussion by gzqcason** [apache/dolphinscheduler#12374] [Q&A] [answered]
It seems like task can not run separately when workflow is online. When I use right click to run the task node I want, DS will execute all downstream dependent tasks. This issue was discussed in the community, but I did not find the description in the updated document of version 3.1.0. Does anyone know how to implement this function in V.3.0.1, or tell me whether the V.3.1.0 fixes this problem? tks a lot.

工作流是上线状态下似乎无法单独运行单节点任务。在任务上使用右键的运行，会执行下游所有的依赖任务。社区中讨论过这个问题，但是我没有在3.1.0版本的更新文档中找到说明。有人知道使用3.0.1版本如何实现么，或者告知一下3.1.0版本中是否修复了这个问题，谢谢！

**Comment by DarkAssassinator** [comment, marked as answer]
hi @gzqcason, now DS did not support, but plan to support it in Q4.
Please refer to https://docs.qq.com/doc/DTHNvbXJwR3VDTHNs
![image](https://user-images.githubusercontent.com/20518339/195864306-89fa7d13-f8bf-4220-bcca-6b359dbc2fce.png)

**Reply by gzqcason** [reply]
Got it, thank you for your reply, BR
